app/assets/javascripts/components/overview/statistics_update_info.jsx

```diff
@@ -49,33 +31,30 @@ const StatisticsUpdateInfo = createReactClass({
   render() {
     const course = this.props.course;
 
-    if ((Features.wikiEd && !course.ended) || !course.updates.last_update) {
+    if (Features.wikiEd && !course.ended) {
       return <div />;
     }
 
-    // Render Modal
+    const [lastUpdateMessage, nextUpdateMessage, isNextUpdateAfter] = this.getUpdateTimesArray();
+
     if (this.state.showModal) {
       return (
         <StatisticsUpdateModal
           course={course}
-          getUpdateTimesInformation={this.getUpdateTimesInformation}
+          isNextUpdateAfter={isNextUpdateAfter}
+          nextUpdateMessage={nextUpdateMessage}
           toggleModal={this.toggleModal}
         />
       );
     }
 
-    const updateTimesInformation = this.getUpdateTimesInformation();
-    let updateTimesMessage = '';
-    if (updateTimesInformation !== null) {
-      const [lastSuccessfulUpdateMessage, nextUpdateMessage, isNextUpdateAfter] = updateTimesInformation;
-      updateTimesMessage = isNextUpdateAfter ? `${lastSuccessfulUpdateMessage}. ${nextUpdateMessage}. ` : `${lastSuccessfulUpdateMessage}. `;
-    }
+    const updateTimesMessage = isNextUpdateAfter ? `${lastUpdateMessage} ${nextUpdateMessage} ` : `${lastUpdateMessage} `;
 
-    // Render update time information along with 'See More' link to open modal
+    // Render update time information and if some updates were made a 'See More' link to open modal
     return (
       <div className="statistics-update-info pull-right mb2">
         <small>
-          { updateTimesMessage }<a onClick={this.toggleModal} href="#">{I18n.t('metrics.update_statistics_link')}</a>
+          {updateTimesMessage} {course.updates.last_update !== null && <a onClick={this.toggleModal} href="#">{I18n.t('metrics.update_statistics_link')}</a>}
```

Comment:
I thought I made the modal link impossible to show before the first update with this.

Anyway, I realized that modal can be useful even if there are no updates yet, at least for the late update info, so I put it back and made it work without updating data.
A link for modal is not shown before enqueued, it could work with as it is, but it will bring no extra information.

It is a bit tricky to replicate enqueued, but not updated situation and I hope I didn't miss anything.

Also, I extracted some code from modal to Utils.